src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/domain/Vilkår.kt

```diff
@@ -64,7 +63,9 @@ data class Vilkår(
         tom: LocalDate,
     ) {
         when (type) {
-            VilkårType.PASS_BARN -> {
+            VilkårType.PASS_BARN,
+            VilkårType.FASTE_UTGIFTER,
+            -> {
                 validerFørsteOgSisteDagIValgtMåned(fom, tom)
```

Comment:
Denne valideringen gjelder ikke for boutgifter tror jeg. De faste utgiftene trenger ikke være første til siste dag i en kalendermåned

Reply:
Litt usikker selv faktisk, men ut i fra skissene ser det ut som faste utgifter er starten og slutten av måned, mens midlertidig utgifter er valgfri dag. Vi kan ta det opp med Marie og Lars på mandag.
https://www.figma.com/design/a6qlyarKG3xG2A1yjkZzMz/Bolig-og-overnatting---TS-sak?node-id=3002-3758&p=f&t=wuxT2lTvAUzz26nj-0